bin/move-to-go

```diff
@@ -211,4 +212,5 @@ class MoveToGoCommandLine < Thor
     end
 end
 
+puts "Version: #{Gem.loaded_specs["move-to-go"].version}"
```

Comment:
Ja, det var tanken. 

Så om den lokala move-to-go är äldre än den som finns på rubygems så ska det skrivas ut och sen bör move-to-go avslutats. 

Om man verkligen vill köra den gamla versionen så ska man kunna det, antingen med en flagga till move-to-go (som gör att kontrollen mot rubygems inte görs) eller genom att skicka versionsnr till gem enligt 
http://stackoverflow.com/questions/4061596/how-can-i-call-an-older-version-of-a-gem-from-the-commandline